### ISSUE TITLE
Can't use ngx-mqtt@5.4.0 with angular 5

### ISSUE THREAD

**Issue by utkarshns** [sclausen/ngx-mqtt#90]
- [x] **I have searched for similar issues in this repository, but couldn't find one.**
- [x] **I have read the README and have a basic understanding how angular works.**
## I'm submitting a...

- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report  
- [ ] Feature request

## Current behavior

Can't use ngx-mqtt 5.4.0 with Angular 5 (a new project).
The project builds just fine but I get the following error in the browser log.
```
AppComponent_Host.ngfactory.js? [sm]:1 ERROR Error: StaticInjectorError(AppModule)[AppComponent -> MqttService]: 
  StaticInjectorError(Platform: core)[AppComponent -> MqttService]: 
    NullInjectorError: No provider for MqttService!
    at _NullInjector.get (core.js:1003)
    at resolveToken (core.js:1301)
    at tryResolveToken (core.js:1243)
    at StaticInjector.get (core.js:1111)
    at resolveToken (core.js:1301)
    at tryResolveToken (core.js:1243)
    at StaticInjector.get (core.js:1111)
    at resolveNgModuleDep (core.js:10896)
    at NgModuleRef_.get (core.js:12129)
    at resolveDep (core.js:12619)
```

## Expected behavior

Tried 5.4.0 and 5.2.0, both produce the same error.

## Minimal reproduction of the problem with instructions

app.component.ts: [gist](https://gist.github.com/utkarshns/9bf68ca9adf031baa98a6c450e6bb16b)
Create a new angular 5 app and try to do a basic implementation.

## Environment

<pre><code>
ngx-mqtt version: 5.4.0 / 5.2.0

- Broker 
Mosquitto

**Comment by utkarshns** [comment]
Hello,
I did the exact same thing. Went back and read the readme.md
here is another implementation:
[https://gist.github.com/utkarshns/4c6d4e283c391d758c8d050ffd694ac6](https://gist.github.com/utkarshns/4c6d4e283c391d758c8d050ffd694ac6)

**Comment by sclausen** [comment]
If that's exactly what you copied, the app can't run. You missed a bunch of essentials. Your NgModule has no class, no BrowserModule import, not bootstrap and so on.
PLLLLEEEEAASSE post a WORKING reproduction of your issue.
This includes your package.json and package-lock.json

**Comment by utkarshns** [comment]
It now works, I fixed the error with NgModules and ngx-mqtt 5.4.0 works with Angular 5.
Thanks for the speedy responses.
